app/src/main/res/values/strings.xml

```diff
@@ -164,20 +108,18 @@
 
     <string name="connection_info">Connection info</string>
     <string name="status_title">Wi-Fi info</string>
-    <string name="status_ip_4">IPV4: %s</string>
+    <string name="status_ip_4">IPv4: %s</string>
     <string name="status_ssid">SSID: %s</string>
-    <string name="status_bssid">BSSID: %s</string>
-    <string name="status_band">Band: %s</string>
-    <string name="status_channel">Channel: %s</string>
+    <string name="status_bssid">@string/bssid</string>
```

Comment:
Remove?

Reply:
I saw that you have the same strings repeated for different views, so I just reused the main ones... What do you mean by Remove?

Reply:
Remove status_bssid and use bssid? I don't think we need aliases.

Reply:
Sure go ahead